helm/sematic-server/templates/deployment.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "sematic-server.fullname" . }}
+  labels:
+    {{- include "sematic-server.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "sematic-server.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "sematic-server.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "sematic-server.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Comment:
The tag is the placeholder for the docker format. The value can be a SHA also

Reply:
The format of ones with a sha is:
`123456789876.dkr.ecr.us-west-2.amazonaws.com/sematic@sha256:ef8fddb7a6fdae2934c77fc09f8c2e2acbde7cf5b5d0b378073aaef5272aa348`, so it's like `<repo>@sha256:<the hash>` as opposed to `<repo>:<the tag>`. But I suppose you could always use `<repo>@sha256` as `.Values.image.repository` and `<the hash>` as `.Values.image.tag`?

Reply:
Do we really need to even parse the image/tag/digest ourselves?  I think it would best if we just take `.Values.image` as a single string, and write that in directly here with a fallback default, and let users handle the registry/image/tag/digest themselves.